tests/e2e/upgrade_test.go

```diff
@@ -385,6 +418,16 @@ var _ = Describe("Upgrade", Label(tests.LabelUpgrade, tests.LabelNoOpenshift), O
 				return err
 			}, timeout).ShouldNot(HaveOccurred())
 		})
+		By("getting the operator info", func() {
+			pod, err := env.GetOperatorPod()
+			Expect(err).NotTo(HaveOccurred())
+			GinkgoWriter.Println("image used for operator", pod.Spec.Containers[0].Image)
+			lgs, err := env.GetPodLogs(pod.Namespace, pod.Name)
+			Expect(err).NotTo(HaveOccurred())
+			lines := strings.Split(lgs, "\n")
+			Expect(lines).ShouldNot(BeEmpty())
+			GinkgoWriter.Println(lines[0])
```

Comment:
I would add here a caption of what we are printing.
Something like
`GinkgoWriter.Println("pod logs", lines[0])`